Signal/src/ViewControllers/OWSBackupExportViewController.m

```diff
@@ -0,0 +1,311 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSBackupExportViewController.h"
+#import "OWSBackup.h"
+#import "OWSProgressView.h"
+#import "Signal-Swift.h"
+#import "ThreadUtil.h"
+#import <SignalMessaging/AttachmentSharing.h>
+#import <SignalMessaging/Environment.h>
+#import <SignalMessaging/SignalMessaging-Swift.h>
+#import <SignalMessaging/UIColor+OWS.h>
+#import <SignalMessaging/UIFont+OWS.h>
+#import <SignalMessaging/UIView+OWS.h>
+#import <SignalServiceKit/MIMETypeUtil.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OWSBackupExportViewController () <OWSBackupDelegate>
```

Comment:
The backup export & import VCs share a lot in common.  We could extract a common base class, or not.